src/partials/about.html

```diff
@@ -42,24 +42,21 @@ <h3 class="about-list-header">04</h3>
         </li>
       </ul>
       <div class="about-picture">
-        <picture>
-          <source srcset="
-              ../img/memoji.png 1x,
-              ../img/memoji@2x.png 2x
-            " />
-          <img class="about-picture-img" src="../img/memoji.png" alt="Smiling girl pointing her finger" />
-          <source
-            srcset="
-              ../img/memoji.png 1x,
-              ../img/memoji@2x.png 2x
-            "
+          <img class="about-picture-mobile"
```

Comment:
look again na prop-leave-app section:
![prop-leave-app-img-html](https://github.com/user-attachments/assets/c420336e-d885-4950-aebe-3d769b0b9ca8)

Reply:
why do you add 3 different images? there is only one img